db/downloads_schema.rb

```diff
@@ -0,0 +1,29 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.1].define(version: 0) do
+  create_schema "_timescaledb_cache"
+  create_schema "_timescaledb_catalog"
+  create_schema "_timescaledb_config"
+  create_schema "_timescaledb_debug"
+  create_schema "_timescaledb_functions"
+  create_schema "_timescaledb_internal"
+  create_schema "timescaledb_experimental"
+  create_schema "timescaledb_information"
+  create_schema "toolkit_experimental"
```

Comment:
This will be hidden and recreated by enabling the extension. Already fixed it in the `timescaledb` gem.

Reply:
How to achieve this?

Reply:
I was missing to merge this: https://github.com/jonatas/timescaledb/pull/66

Going to release a new version. So, when you add the timescaledb gem it will be able to skip the extension in case it's not available.

Reply:
`timescaledb` is not used in this PR. Should I add it as well already?

Reply:
Yes! Tested now, it works as expected. I'm adding now.